app/services/submit_application.rb

```diff
@@ -10,7 +10,12 @@ def initialize(application_form)
 
   def call
     ActiveRecord::Base.transaction do
-      application_form.update!(submitted_at: Time.zone.now)
+      if application_form.apply_again? && enough_references_have_been_provided? && FeatureFlag.active?('move_edit_by_to_application_form')
```

Comment:
This should happen irrespective of the feature flag because we need it to work whether it's on or not:

```suggestion
      if application_form.apply_again? && enough_references_have_been_provided?
```